src/Twig/GdprExtension.php

```diff
@@ -21,7 +21,7 @@ public function hasPermission(Environment $env, string $permission): bool
     {
         $function = $env->getFunction('sylius_plus_rbac_has_permission');
 
-        if ($function !== null && is_callable($function->getCallable())) {
+        if (is_object($function) && is_callable($function->getCallable())) {
```

Comment:
i would suggest this instead
```suggestion
        if ($function instanceof TwigFunction && is_callable($function->getCallable())) {
```